lib/git-runner.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+var fs = require('fs');
+
+module.exports = GitRunner;
+
+function GitRunner(config, builderOpts, commandRunner, logger) {
+  this.git = config.git;
+  this.githubOrg = builderOpts.githubOrg;
+  this.repoDir = builderOpts.repoDir;
+  this.sitePath = builderOpts.sitePath;
+  this.commandRunner = commandRunner;
+  this.logger = logger;
+}
+
+GitRunner.prototype.prepareRepo = function(branch) {
+  var gitRunner = this;
+
+  return new Promise(function(resolve) {
+    fs.exists(gitRunner.sitePath, function(exists) {
```

Comment:
Right, then `fs.exists` just passes false, and the repo is cloned instead of synced.